can-simple-observable-test.js

```diff
@@ -32,3 +32,34 @@ QUnit.test('basics', function(){
 
     QUnit.equal(canReflect.getValue(obs), 'four', 'getValue after offValue');
 });
+
+QUnit.test("log", function(assert) {
```

Comment:
normally we use `dev.log`.  https://github.com/canjs/can-define/blob/21787bb1021c464cdff4b8d40c40c9bfe045526a/define-test.js#L1373  @bmomberger-bitovi can probably point you to the most modern way of testing this.

Reply:
I would recommend routing all of your logging through `can-log/dev`, which has all of the features of can-log in dev but noops in production.  You should still wrap calls to logging in steal-remove-start/steal-remove end for file size reasons.

However, I only wrote verifiers for warnings and errors in `can-test-helpers/dev`, so unless @m-mujica wants to go back and extend it to create a "willLog" helper (which is a one-line addition plus some copypasted comments), it won't be of any value here.